graylog2-web-interface/src/pages/SidecarStatusPage.jsx

```diff
@@ -38,8 +40,13 @@ class SidecarStatusPage extends React.Component {
     SidecarsActions.getSidecar(this.props.params.sidecarId).then(sidecar => this.setState({ sidecar }));
   };
 
+  reloadCollectors() {
```

Comment:
As this is an ES6 class, this should use an arrow function, to ensure `this` is properly bound, even if the function calling it is not autobound by React.

From https://reactjs.org/docs/react-without-es6.html#autobinding:

> If you’d rather play it safe, you have a few options:
>  - Bind methods in the constructor.
>  - Use arrow functions, e.g. `onClick={(e) => this.handleClick(e)}`.
>  - Keep using `createReactClass`.